src/EditorFeatures/Test/Preview/PreviewWorkspaceTests.cs

```diff
@@ -125,52 +125,15 @@ public async Task TestPreviewServices()
         {
             using var previewWorkspace = new PreviewWorkspace(EditorTestCompositions.EditorFeatures.GetHostServices());
             var service = previewWorkspace.Services.GetService<ISolutionCrawlerRegistrationService>();
-            Assert.IsType<PreviewSolutionCrawlerRegistrationServiceFactory.Service>(service);
+            var registrationService = Assert.IsType<SolutionCrawlerRegistrationService>(service);
+            Assert.False(registrationService.Register(previewWorkspace));
 
             var persistentService = previewWorkspace.Services.SolutionServices.GetPersistentStorageService();
 
             await using var storage = await persistentService.GetStorageAsync(SolutionKey.ToSolutionKey(previewWorkspace.CurrentSolution), CancellationToken.None);
             Assert.IsType<NoOpPersistentStorage>(storage);
         }
 
-        [WorkItem(923196, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/923196")]
-        [WpfFact]
-        public async Task TestPreviewDiagnostic()
```

Comment:
pull tagger test is below.  this one was testing the push/solution-crawler version.